### PR TITLE
[SPARK-30093][SQL] Improve error message for creating view

### PR DESCRIPTION
**Pull request by amanomer**
### What changes were proposed in this pull request?
Improved error message while creating views.

### Why are the changes needed?
Error message should suggest user to use TEMPORARY keyword while creating permanent view referred by temporary view.
https://github.com/apache/spark/pull/26317#discussion_r352377363

### Does this PR introduce any user-facing change?
No

### How was this patch tested?
Updated test case.

### CODE REVIEW

**Comment by amanomer** [comment]
cc @gatorsmile

**Comment by amanomer** [comment]
cc @srowen @cloud-fan

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala [hunk 1]
```diff
@@ -162,7 +162,7 @@ case class CreateViewCommand(
             if sparkSession.sessionState.catalog.isTemporaryTable(ident) =>
             // temporary views are only stored in the session catalog
             throw new AnalysisException(s"Not allowed to create a permanent view $name by " +
-              s"referencing a temporary view $ident")
+              s"referencing a temporary view $ident (use TEMPORARY keyword)")
```
Comment (cloud-fan) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala]: nit: `... referencing a temporary view $ident. Please create a temp view instead by CREATE TEMP VIEW`

Comment (amanomer) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala]: Done.

**Comment by amanomer** [comment]
Jenkins, test this please.

**Comment by amanomer** [comment]
cc @cloud-fan

**Comment by cloud-fan** [comment]
ok to test

**Comment by amanomer** [comment]
cc @cloud-fan

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Pull request closed without merging**